src/content/refi/index.md

```diff
@@ -0,0 +1,80 @@
+alt: ""
+summaryPoint1: An alternative economic system built on regenerative principles
+summaryPoint2: An attempt to harness Ethereum to solve global-level coordination crises such as climate change
+summaryPoint3: A tool to drastically scale ecological benefit assets like verifed carbon credits
+---
+
+## What is ReFi? {#what-is-refi}
+
+**Regenerative finance (ReFi)** is a set of tools and ideas facilitated by Ethereum that create economies which are regenerative in nature rather than extractive or exploitative. Eventually, extractive systems deplete the resources available and collapse; without regenerative mechanisms, they lack resilience. ReFi operates on the assumption that the creation of monetary value must be decoupled from the unsustainable extraction of resources from our planet and communities.
+
+Instead, ReFi aims to solve large-scale issues like climate change as well as more regionally concentrated crises—environmental, communal, or social—by creating regenerative cycles. These systems create value for participants while simultaneously benefitting our ecosystem and/or communities. In economic parlance, regenerative financial systems incentivize the production of positive externalities.
+
+ReFi leverages one of capitalism’s strong suits—its ability to rapidly deploy capital toward activities that offer a return to stakeholders—in the best interests of said stakeholders, our planet, and future generations. It draws on the theory of regenerative economics pioneered by John Fullerton of the [Capital Institute](https://capitalinstitute.org) to work toward the realization of eight interconnected principles that underlie systemic health:
+
+![Eight interconnected principles](../../assets/use-cases/refi-regenerative-economy-diagram.png)
+
+To do this, ReFi projects utilize [smart contracts](/developers/docs/smart-contracts/) and tools pioneered in the [decentralized finance (DeFi)](/defi/) space to incentivize regenerative behaviors and facilitate large-scale collaboration.
+
+ReFi also overlaps with the [decentralized science (DeSci)](/desci/) movement, which uses Ethereum as a platform to finance, create, review, credit, store, and disseminate scientific knowledge. Such processes for distributed knowledge generation play a critical role in developing verifiable standards and practices for implementing and monitoring regenerative activities like planting trees, removing plastic from the ocean, or restoring a degraded ecosystem.
+
+## The current state of ReFi {#refi-current-state}
+
+ReFi is in its early stages, and has so far found its best fit in projects that seek to shape financial incentives toward the meeting of United Nations [Sustainable Development Goal 13 on Climate Change](https://sdgs.un.org/goals/goal13). Tokenization of carbon credits from the **[voluntary carbon market (VCM)](https://climatefocus.com/so-what-voluntary-carbon-market-exactly/)** is currently the most prominent use case.

```

Comment:
```suggestion
ReFi is still very young, and so far it has found its best fit in projects related to carbon markets. Tokenization of carbon credits from the **[voluntary carbon market (VCM)](https://climatefocus.com/so-what-voluntary-carbon-market-exactly/)** is currently the most prominent use case.

```

Reply:
I like that you have removed one link to make it easier on the reader. I think its still a bit confusing because its very abstract.

Reply:
```suggestion
**[Voluntary carbon markets (VCM)](https://climatefocus.com/so-what-voluntary-carbon-market-exactly/)** let polluting companies buy credits from projects that reduce or remove greenhouse gases in the atmosphere to balance out their own emissions. This tokenization can be easily tracked, audited and managed on Ethereum.
```

Reply:
i have rewritten this part to focus on explaining the problem - what are carbon markets in the first place, we should not assume the reader knows